src/shared/ui/StatusBar/index.tsx

```diff
@@ -0,0 +1,10 @@
+import React, { FC } from 'react';
+import { StatusBar as BaseStatusBar, Platform } from 'react-native';
+
+const isIOS = Platform.OS === 'ios';
```

Comment:
Please move the constant to /shared/lib/constants.ts and name it IS_IOS. You can also create IS_ANDROID accordingly.